sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -132,7 +132,7 @@ namespace Azure { namespace Core { namespace Http {
      * @brief When `true`, allows an invalid certificate authority. If this flag is set, the
      * application does not receive a WINHTTP_CALLBACK_STATUS_FLAG_INVALID_CA callback.
```

Comment:
unrelated nit: remove the callback snippet from the doc, it isn't relevant to the caller/user of this option since they aren't using WinHttp directly or care about this.

```suggestion
     * @brief When `true`, allows an invalid certificate authority.
```